app/components/accordion_component.rb

```diff
@@ -0,0 +1,7 @@
+class AccordionComponent < ViewComponent::Base
+  renders_one :header
+
+  def initialize
+    @target_id = "accordion-#{SecureRandom.hex(4)}"
```

Comment:
note mostly for myself, this is 4 bytes of randomness/8 chars of hex, so 2^32 values so not likely to collide

Reply:
Reminds me it'd be good to have a spec for ID presence / uniqueness: e104863